docs/modules/ROOT/pages/api/erc1155.adoc

```diff
@@ -0,0 +1,665 @@
+and <<ERC1155Component-TransferBatch,TransferBatch>> otherwise.
+
+[.contract-item]
+[[ERC1155Component-mint_with_acceptance_check]]
+==== `[.contract-item-name]#++mint_with_acceptance_check++#++(ref self: ContractState, to: ContractAddress, token_id: u256, value: u256, data: Span<felt252>)++` [.item-kind]#internal#
+
+Creates a `value` amount of tokens of type `token_id`, and assigns them to `to`.
+
+Requirements:
+
+- `to` cannot be the zero address.
+- If `to` refers to a smart contract, it must implement `IERC1155Receiver::on_ERC1155_received`
+and return the acceptance magic value.
+
+Emits a <<ERC1155Component-TransferSingle,TransferSingle>> event.
+
+[.contract-item]
+[[ERC1155Component-batch_mint_with_acceptance_check]]
+==== `[.contract-item-name]#++batch_mint_with_acceptance_check++#++(ref self: ContractState, to: ContractAddress, token_ids: Span<u256>, values: Span<u256>, data: Span<felt252>)++` [.item-kind]#internal#
+
+Batched version of <<ERC1155Component-mint_with_acceptance_check,mint_with_acceptance_check>>.
+
+Requirements:
+
+- `to` cannot be the zero address.
+- `token_ids` and `values` must have the same length.
+- If `to` refers to a smart contract, it must implement `IERC1155Receiver::on_ERC1155_batch_received`
+and return the acceptance magic value.
+
+Emits a <<ERC1155Component-TransferBatch,TransferBatch>> event.
+
+[.contract-item]
+[[ERC1155Component-burn]]
+==== `[.contract-item-name]#++burn++#++(ref self: ContractState, from: ContractAddress, token_id: u256, value: u256)++` [.item-kind]#internal#
+
+Destroys a `value` amount of tokens of type `token_id` from `from`.
+
+Requirements:
+
+- `from` cannot be the zero address.
+- `from` must have at least `value` amount of tokens of type `token_id`.
+
+Emits a <<ERC1155Component-TransferSingle,TransferSingle>> event.
+
+[.contract-item]
+[[ERC1155Component-batch_burn]]
+==== `[.contract-item-name]#++batch_burn++#++(ref self: ContractState, from: ContractAddress, token_ids: Span<u256>, values: Span<u256>)++` [.item-kind]#internal#
+
+Batched version of <<ERC1155Component-burn,burn>>.
+
+Requirements:
+
+- `from` cannot be the zero address.
+- `from` must have at least `value` amount of tokens of type `token_id`.
+- `token_ids` and `values` must have the same length.
+
+Emits a <<ERC1155Component-TransferBatch,TransferBatch>> event.
+
+[.contract-item]
+[[ERC1155Component-set_uri]]
+==== `[.contract-item-name]#++set_uri++#++(ref self: ContractState, uri: ByteArray)++` [.item-kind]#internal#
+
+Sets a new URI for all token types, by relying on the token type ID
+substitution mechanism defined in the ERC1155 standard.
+
+See https://eips.ethereum.org/EIPS/eip-1155#metadata.
+
+By this mechanism, any occurrence of the `\{id\}` substring in either the
+URI or any of the values in the JSON file at said URI will be replaced by
+clients with the token type ID.
+
+For example, the `https://token-cdn-domain/\{id\}.json` URI would be
```

Comment:
```suggestion
For example, the pass:[<code>https://token-cdn-domain/\{id\}.json</code>] URI would be
```
I'm not a fan of this being clickable. I'd change the other other example in this sentence if you agree